### PR TITLE
Add gcr.io/k8s-staging-descheduler

### PR DESCRIPTION
**Pull request by seanmalloy**
This is for https://github.com/kubernetes-incubator/descheduler

### CODE REVIEW

**Comment by seanmalloy** [comment]
@aveshagarwal and @ravisantoshgudimetla please verify that I have your email addresses correct in `groups/groups.yaml`.

File: groups/groups.yaml [hunk 1]
```diff
@@ -108,6 +108,10 @@ groups:
       - detiber@gmail.com
       - vince@vincepri.com
       - hi@amycod.es
+  - email-id: k8s-infra-staging-descheduler@kubernetes.io
+    members:
+      - ravisantoshgudimetla@gmail.com
```
Comment (thockin) [inline groups/groups.yaml]: @dims @spiffxp Do we auto-add folks like yourselves to all groups, or do we expect that to be done here, or do we not care?

File: infra/gcp/ensure-staging-storage.sh [hunk 2]
```diff
@@ -43,6 +43,7 @@ STAGING_PROJECTS=(
     cip-test
     cluster-api
     cluster-api-aws
+    descheduler
```
Comment (thockin) [inline infra/gcp/ensure-staging-storage.sh]: alphabetize please

File: groups/groups.yaml (hunk 1 above)
Comment (dims) [reply groups/groups.yaml]: @spiffxp what do you lean towards? i am ok either way

File: infra/gcp/ensure-staging-storage.sh (hunk 2 above)
Comment (seanmalloy) [reply infra/gcp/ensure-staging-storage.sh]: Fixed

File: groups/groups.yaml (hunk 1 above)
Comment (dims) [reply groups/groups.yaml]: @seanmalloy for now, please add me and Aaron to the list of members for better clarity on who has access

**Comment by dims** [comment]
@seanmalloy - i am guessing this falls under the sig-scheduler? Can you please get an ok from the SIG leads?

File: groups/groups.yaml (hunk 1 above)
Comment (seanmalloy) [reply groups/groups.yaml]: Fixed. I added @dims and @spiffxp to the list.

**Comment by seanmalloy** [comment]
@dims yes the descheduler is part of sig-scheduling. Looks like @bsalamat and @k82cn are the sig-scheduling leads and would need to review this PR.

Would also be great if @aveshagarwal and @ravisantoshgudimetla could review as they are the maintainers for descheduler.

**Comment by k82cn** [comment]
that's ok to me :)

**Comment by aveshagarwal** [comment]
My email id is: avesh.ncsu@gmail.com

**Comment by dims** [comment]
@seanmalloy please update with Avesh's email address?

**Comment by seanmalloy** [comment]
I have corrected the email for @aveshagarwal. Let me know if any other changes are required or if you want me to squash commits.

**Comment by dims** [comment]
yes, please squash the commits.

**Comment by seanmalloy** [comment]
@dims commits have been squashed.

**Comment by ravisantoshgudimetla** [comment]
LGTM from me :+1:

**Comment by aveshagarwal** [comment]
Thanks Sean.

**Comment by dims** [comment]
/assign

will create the stuff needed later today

**Comment by dims** [comment]
Google group created

```
2019/07/17 16:41:23 > Successfully updated group settings for k8s-infra-staging-cluster-api-aws@kubernetes.io to allow external members and other security settings
2019/07/17 16:41:23 Trying to create group: k8s-infra-staging-descheduler@kubernetes.io
2019/07/17 16:41:24 > Successfully created group k8s-infra-staging-descheduler@kubernetes.io
2019/07/17 16:41:25 > Successfully updated group settings for k8s-infra-staging-descheduler@kubernetes.io to allow external members and other security settings
2019/07/17 16:41:26 Added ravisantoshgudimetla@gmail.com to k8s-infra-staging-descheduler@kubernetes.io as a MEMBER
2019/07/17 16:41:27 Added avesh.ncsu@gmail.com to k8s-infra-staging-descheduler@kubernetes.io as a MEMBER
2019/07/17 16:41:28 Added spiffxp@google.com to k8s-infra-staging-descheduler@kubernetes.io as a MEMBER
2019/07/17 16:41:29 Added davanum@gmail.com to k8s-infra-staging-descheduler@kubernetes.io as a MEMBER
```

**Comment by dims** [comment]
GCS bucket created

```
[dims@dims-vmware-laptop 16:46] ~/go/src/k8s.io/k8s.io ⟩ infra/gcp/ensure-staging-storage.sh descheduler
descheduler
Ensuring project exists: k8s-staging-descheduler
Create in progress for [https://cloudresourcemanager.googleapis.com/v1/projects/k8s-staging-descheduler].
Waiting for [operations/cp.8415770674804595411] to finish...done.
billingAccountName: billingAccounts/018801-93540E-22A20E
billingEnabled: true
name: projects/k8s-staging-descheduler/billingInfo
projectId: k8s-staging-descheduler
Empowering k8s-infra-staging-descheduler@kubernetes.io as project viewers
Updated IAM policy for project [k8s-staging-descheduler].
bindings:
- members:
  - user:davanum@gmail.com
  role: roles/owner
- members:
  - group:k8s-infra-staging-descheduler@kubernetes.io
  role: roles/viewer
etag: BwWN5pbrlKU=
version: 1
Enabling the container registry API
Operation "operations/acf.02d4f733-baf0-47d5-96e8-a7aacf586959" finished successfully.
Ensuring the registry exists and is readable
Using default tag: latest
latest: Pulling from pause
a3ed95caeb02: Pull complete
4964c72cd024: Pull complete
Digest: sha256:a78c2d6208eff9b672de43f880093100050983047b7b0afe0217d3656e1b0d5f
Status: Downloaded newer image for k8s.gcr.io/pause:latest
The push refers to repository [gcr.io/k8s-staging-descheduler/ceci-nest-pas-une-image]
5f70bf18a086: Layer already exists
e16a89738269: Pushed
latest: digest: sha256:ec3ca3ee90e4dafde96c83232b30f17b5e8992ff35479d0661b8f4ff2f21bf74 size: 938
Digests:
- gcr.io/k8s-staging-descheduler/ceci-nest-pas-une-image@sha256:ec3ca3ee90e4dafde96c83232b30f17b5e8992ff35479d0661b8f4ff2f21bf74
  Associated tags:
 - latest
Tags:
- gcr.io/k8s-staging-descheduler/ceci-nest-pas-une-image:latest
Deleted [gcr.io/k8s-staging-descheduler/ceci-nest-pas-une-image:latest].
Deleted [gcr.io/k8s-staging-descheduler/ceci-nest-pas-une-image@sha256:ec3ca3ee90e4dafde96c83232b30f17b5e8992ff35479d0661b8f4ff2f21bf74].
Enabling Bucket Policy Only for gs://artifacts.k8s-staging-descheduler.appspot.com...
Empowering GCR admins
Updated IAM policy for project [k8s-staging-descheduler].
bindings:
- members:
  - serviceAccount:service-1009880777024@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:davanum@gmail.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-descheduler@kubernetes.io
  role: roles/viewer
etag: BwWN5pkklmo=
version: 1
Empowering k8s-infra-staging-descheduler@kubernetes.io to GCR
Enabling the GCS API
Operation "operations/acf.506ce52b-a372-4aa6-ab9c-59b7d6eb10c1" finished successfully.
Ensuring the bucket exists and is world readable
Creating gs://k8s-staging-descheduler/...
Enabling Bucket Policy Only for gs://k8s-staging-descheduler...
Ensuring the bucket has retention of 30d
Setting Retention Policy on gs://k8s-staging-descheduler/...
Ensuring the bucket has auto-deletion of 60 days
Setting lifecycle configuration on gs://k8s-staging-descheduler/...
Empowering GCS admins
Updated IAM policy for project [k8s-staging-descheduler].
bindings:
- members:
  - serviceAccount:service-1009880777024@containerregistry.iam.gserviceaccount.com
  role: roles/editor
- members:
  - user:davanum@gmail.com
  role: roles/owner
- members:
  - group:k8s-infra-artifact-admins@kubernetes.io
  - group:k8s-infra-staging-descheduler@kubernetes.io
  role: roles/viewer
etag: BwWN5prE614=
version: 1
Empowering k8s-infra-staging-descheduler@kubernetes.io to GCS
Done
```

**Pull request merged**